templates/etc/systemd/system/notification.service

```diff
@@ -5,7 +5,7 @@ After=network.target
 [Service]
 Type=simple
 WorkingDirectory=/data/www/notification-service
-ExecStart=/usr/bin/nodejs /data/www/notification-service/lib/index.js
+ExecStart=node /data/www/notification-service/lib/index.js
```

Comment:
The reason we have to use nodejs (instead of node) on Debian is because of a name conflict with another package.